src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1327,7 +1327,7 @@ def update_resource_group(instance, tags=None):
 
 def export_group_as_template(
         cmd, resource_group_name, include_comments=False, include_parameter_default_value=False, resource_ids=None, skip_resource_name_params=False, skip_all_params=False):
-    """Captures a resource group as a template.
+    """Capture a resource group as a template.
```

Comment:
In fact, they are just comments in the code which will not affect the help message. However, these changes themselves look good~